web/src/features/App/api/getLicense.ts

```diff
@@ -31,11 +37,12 @@ export const getLicense = async (appSlug: string) => {
   }
 };
 
-export const useLicense = (params: string) => {
-  return useQuery(["license", params], () => getLicense(params), {
-    /// might want to disable the fetch on window focus for this one
-    refetchInterval: 5000,
-  });
+export const useLicense = () => {
+  const { selectedApp } = useSelectedApp();
```

Comment:
Rather than having components pass down the `selectedApp`, opting to use the custom hook for it.